lib/msf/base/serializer/readable_text.rb

```diff
@@ -768,8 +768,9 @@ def self.dump_sessions_verbose(framework, opts={})
       sess_checkin = "<none>"
       sess_registration = "No"
 
+      # to_s is required incase something bugged earlier and .platform is nil
       if session.respond_to?(:platform)
-        sess_type << " " + session.platform
+        sess_type << " " + session.platform.to_s
```

Comment:
Arguably you shouldn't be appending anything at all if `platform` is `nil`.

Reply:
Perfect, thanks. I'd just do `if session.respond_to?(:platform) && session.platform`, though. The value is truthy enough to be checked that way without the negative logic.